### DISCUSSION TITLE
Insert questions into a book or a page of text

### DISCUSSION THREAD

**Discussion by BlacknBlueBerry** [srobotta/moodle-tiny_cloze#23] [Q&A]
Dear team,
I am still not quite sure whether I have understood the plugin.
I can apparently use it to create questions via an editor (also only in the question collection).
But I have not yet found a way to insert these created questions where (for example in a book or a text page).
Is this even possible, or have I just overlooked something?

**Comment by marcusgreen** [comment]
This might help
https://moodle.org/plugins/filter_embedquestion

**Reply by BlacknBlueBerry** [reply]
Thank you very much!
I must have overlooked that, as I was only looking for the plugin on 4.1.
Thank you very much!
I assume that the old plugin here still works on 4.1?
And i think it doesn't work on the TinyMce Editor?

**Comment by marcusgreen** [comment]
I don't know much about those things apart from the fact they exist. I suspect it will work with the "new" TinyMCE editor, and if it doesn't it will very soon,  because the UK Open University is going down that route and the brains behind it Tim Hunt is very, very keen on that sort of thing.
